ionoscloud/resource_k8s_node_pool_test.go

```diff
@@ -202,3 +224,43 @@ resource "ionoscloud_k8s_node_pool" "terraform_acctest" {
   storage_size      = 40
   public_ips        = [ "%s", "%s", "%s" ]
 }`
+
+const testAccCheckk8sNodepoolConfigLan = `
+resource "ionoscloud_datacenter" "terraform_acctest" {
+  name        = "terraform_acctest_lan"
+  location    = "us/las"
+  description = "Datacenter created through terraform"
+}
+
+resource "ionoscloud_lan" "terraform_acctest" {
+  datacenter_id = "${ionoscloud_datacenter.terraform_acctest.id}"
+  public = false
+  name = "terraform_acctest_lan"
+}
+
+resource "ionoscloud_k8s_cluster" "terraform_acctest" {
+  name        = "terraform_acctest_lan"
+  k8s_version = "1.20.6"
+  maintenance_window {
+    day_of_the_week = "Monday"
+    time            = "09:00:00Z"
+  }
+}
+
+resource "ionoscloud_k8s_node_pool" "terraform_acctest" {
+  name              = "%s"
+  datacenter_id     = ionoscloud_datacenter.terraform_acctest.id
+  k8s_cluster_id    = ionoscloud_k8s_cluster.terraform_acctest.id
+  k8s_version       = ionoscloud_k8s_cluster.terraform_acctest.k8s_version
+  lans {
+    id   = ionoscloud_lan.terraform_acctest.id
+    dhcp = false
```

Comment:
Please include a test for checking the update from `dhcp = true` to `dhcp = false` to make sure everything is fine in the `Update` function with regards to `fetching` the correct value for the dhcp field.